app/src/main/res/values-it/strings.xml

```diff
@@ -1,25 +1,25 @@
 <resources>
-    <string name="app_name">Simple Calculator</string>
-    <string name="app_launcher_name">Calculator</string>
+    <string name="app_name">Calcolatrice semplice</string>
```

Comment:
Keep this consistent please. Either translate app names in every app, or nowhere :) Thanks

Reply:
The second commit will fix this. In all other applications the app name is not translated, only the app launcher name.